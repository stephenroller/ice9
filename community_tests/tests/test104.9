# type foo = int[2] var a : foo[2] a[0] := 0;				fail

type foo = int[2]
var a : foo[2]

a[0] 
	:= 
	0
	;