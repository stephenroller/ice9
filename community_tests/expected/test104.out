line 7: incompatible types to binary operator :=